components/Press/PressLinks/PressLinks.tsx

```diff
@@ -25,13 +25,15 @@ function PressLinks() {
               value={region}
               className={styles.tabsContent}
             >
-              {Articles[region].map(link => (
-                <li key={`GroupLink_${link.url}`}>
-                  <OutboundLink href={link.url} analyticsEventLabel="Press Article">
-                    {link.title}
-                  </OutboundLink>
-                </li>
-              ))}
+              {(Articles as Record<string, any>)[region].map(
```

Comment:
ugh Object.keys is [annoying](https://fettblog.eu/typescript-better-object-keys/)

Will you please create a utility for Sindre's `objectKeys` fn ([see](https://github.com/sindresorhus/ts-extras/blob/main/source/object-keys.ts)):
`export const objectKeys = Object.keys as <Type extends object>(value: Type) => Array<ObjectKeys<Type>>;`

then you can do `objectKeys(Articles)` on L22 and NOT need to explicitly add types to anything in this file.